packages/paste-core/components/menu/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,399 @@
+import * as React from 'react';
```

Comment:
These are the new tests, not sure why it's hiding this diff, maybe it's just large enough 🤷‍♀️ @TheSisb